labellab_mobile/lib/screen/project/edit_image/project_edit_image_state.dart

```diff
@@ -16,4 +17,9 @@ class ProjectEditImageState {
   ProjectEditImageState.success({this.image}) {
     isLoading = false;
   }
+
+  ProjectEditImageState.edited({this.image}) {
+    isLoading = false;
```

Comment:
Currently, ```isLoading``` is set only to false. So it's redundant to set it in every constructor. Can move the initialization to the declaration?

Reply:
Noted :+1: